lte/gateway/c/oai/tasks/amf/amf_authentication.cpp

```diff
@@ -595,10 +596,43 @@ int amf_proc_authentication_complete(
     for (idx = 0; idx < amf_ctx->_vector[auth_proc->ksi].xres_size; idx++) {
       if ((amf_ctx->_vector[auth_proc->ksi].xres[idx]) !=
           msg->autn_response_parameter.response_parameter[idx]) {
+        is_xres_validation_failed = true;
         break;
       }
     }
 
+    if (is_xres_validation_failed) {
```

Comment:
add a verbose comment and spec reference on how you are handling the xres validation failure.